nucleus/deploy/client.py

```diff
@@ -159,6 +165,15 @@ def create_model_bundle(
                 model_bundle_name,
             )
 
+        # Prepare cloudpickle for external imports
+        if globals_copy:
+            for module in get_imports(globals_copy):
+                if module.__name__ == cloudpickle.__name__:
+                    # Avoid recursion
+                    # register_pickle_by_value does not work properly with itself
+                    continue
+                cloudpickle.register_pickle_by_value(module)
```

Comment:
@yixu34 did this `cloudpickle.register_pickle_by_value` thing end up working? Thought it didnt?

Reply:
hmm so it works just that the only limitation is you have to import the full module and outside the function?

Reply:
@syandroo given that they say on cloudpickle page: "Note that this feature is still experimental" I wouldn't be very hopeful :)

At least a simple use case of multiple .py files works fine.

I guess the next step is to try on one of the scale ml projects